### DISCUSSION TITLE
run or raise script for firefox

### DISCUSSION THREAD

**Discussion by gvozdemon** [labwc/labwc#2386] [Q&A]
please write someone a working run or raise script for firefox-esr (debian bookworm)

**Comment by johanmalm** [comment]
Is this type of thing no good?

https://labwc.github.io/tips-and-tricks.html#run-or-raise

**Reply by Flrian** [reply]
Should work with a labwc keybind instead now, too:

```
    <keybind key="W-F1">
      <action name="ForEach">
        <query identifier="org.mozilla.firefox"/>
        <then>
          <action name="Raise"/>
          <action name="Focus"/>
        </then>
        <none>
          <action name="Execute" command="gtk-launch org.mozilla.firefox.desktop"/>
        </none>
      </action>
    </keybind>
```

^ Probably have to adjust the identifier and the command, I'm running firefox via flatpak.
